_notices/rsn0030.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 30 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Docker Image Changes in Release v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"
+notice_created: 2023-04-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-04-20
+---
+
+## Overview
+
+RAPIDS is making a number of changes to its end-user Docker images to improve functionality, performance, and maintainability. These changes are targeted to be introduced in the RAPIDS 23.06 release.
+
+While many of these changes should be seamless to users, **some of them are breaking**. Users should review the full list of changes in the associated issue https://github.com/rapidsai/docker/issues/539

```

Comment:
I looked at the preview and this text wasn't being linked. Let's make sure it links.
```suggestion
While many of these changes should be seamless to users, **some of them are breaking**. Users should review the full list of changes in the associated issue: [https://github.com/rapidsai/docker/issues/539](https://github.com/rapidsai/docker/issues/539)

```